components/core/Field.js

```diff
@@ -105,22 +97,20 @@ export default function Field({
   const showError = touched && error;
   const showSuccess = touched && !error;
 
-  const STYLES = React.useMemo(() => {
+  const STYLES_STATUS = React.useMemo(() => {
     if (showError) return STYLES_INPUT_ERROR;
     if (showSuccess) return STYLES_INPUT_SUCCESS;
-    return STYLES_INPUT;
   }, [touched, error]);
 
   const ContainerComponent = containerAs || "div";
   const ErrorWrapper = errorAs || "div";
   return (
     <div>
       <ContainerComponent>
-        <Input inputCss={STYLES} {...props} />
+        <Input inputCss={[STYLES_INPUT, STYLES_STATUS]} {...props} />
       </ContainerComponent>
-      {props.name === "password" && validations ? (
-        <PasswordValidations validations={validations} />
-      ) : (
+      {props.name === "password" && validations(<PasswordValidations validations={validations} />)}
```

Comment:
Getting this error with this line when I try to sign in 
![image](https://user-images.githubusercontent.com/33686587/122109445-3de24f80-cdd2-11eb-94f1-474d5cec134c.png)

Reply:
Thank you @martinalong it should be fixed now